tests/nominal/test_GroupRareLevelsTransformer.py

```diff
@@ -235,7 +235,7 @@ def test_expected_output_no_weight(self, df, expected):
         x = GroupRareLevelsTransformer(columns=["b", "c"], cut_off_percent=0.2)
 
         # set the mappging dict directly rather than fitting x on df so test works with decorators
-        x.non_rare_levels = {"b": ["a", np.nan], "c": ["e", "c", "a"]}
+        x.non_rare_levels = {"b": ["a", None], "c": ["e", "c", "a"]}
```

Comment:
Q: Do the mappings for the other functions in this test suite need updating also? E.g. for "test_expected_output_no_weight_single_row_na" or "test_expected_output_no_weight_single_row_na_category_column" - or is this part of another a refactor?

Reply:
The expected dfs? yeah they will eventually, but was going to leave as out of scope for now and to be done when this transformer gets narwhalified